aiven/client/client.py

```diff
@@ -1530,7 +1576,7 @@ def list_service_tags(self, project, service):
         )
         return self.verify(self.get, path)
 
-    def update_service_tags(self, project, service, tag_updates):
+    def update_service_tags(self, project: str, service: str, tag_updates: Mapping) -> Mapping:
```

Comment:
`tag_updates: Mapping[str, str]`

Reply:
from `AivenCLI._tag_update_body_from_args`, it seems that it might be `None` value as well, so I guess `Mapping[str, Optional[str]]` would be more precise?